qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -321,11 +265,17 @@ def depolarizing_error(param, num_qubits, standard_gates=True):
     probs = [prob_iden] + (num_terms - 1) * [prob_pauli]
     # Generate pauli strings. The order doesn't matter as long
     # as the all identity string is first.
-    paulis = [
-        "".join(tup)
-        for tup in it.product(['I', 'X', 'Y', 'Z'], repeat=num_qubits)
-    ]
-    return pauli_error(zip(paulis, probs), standard_gates=standard_gates)
```

Comment:
This should be update to return the Pauli instructions, since depolarizing error is a special case of Pauli error:
```python
paulis = [
        Pauli("".join(tup))
        for tup in it.product(['I', 'X', 'Y', 'Z'], repeat=num_qubits)
    ]
return QuantumError(zip(paulis, probs))
```

Reply:
OK, I understand you are suggesting to change the spec of the pauli error (including depolarizing error) so that it internally stores `PauliGate` for the cases `num_qubits > 1` whereas it originally stored composition of I/X/Y/ZGates if `standard_gates=True` and `UnitaryGate` if `standard_gates=False`. I think that's a good idea (and this is a good timing). Done at 60908d6